entity-operator-stunnel/image.yaml

```diff
@@ -21,6 +21,11 @@ modules:
   install:
     - name: entity-operator-stunnel
 
+packages:
+  content_sets:
+    x86_64:
+      - rhel-7-server-rpms
+
```

Comment:
it seems we don't need this here. Did you put it just for consistency as we have for other image descriptors?

Reply:
As I understand, we need to define content_sets in any child images of images that installed RPMs. This is so Freshmaker will know to respin images that inherited RPMs containing CVEs from their parent